src/main/java/com/mide/gangsaeng/bannedword/BannedWordServiceImpl.java

```diff
@@ -32,6 +38,25 @@ public List<BannedWord> list() {
         return cache.values().stream().toList();
     }
 
+    @Override
+    public void findBannedWords(String content) {
+        String target = content.toLowerCase();
```

Comment:
why did you call `toLowerCase` ?

Reply:
Because client can write dirty words by mixing uppercase and lowercase letters.

Reply:
Then your word also should be lower case alphabet when created?

Reply:
> Then your word also should be lower case alphabet when created?

Oh yes, that's a good point. I fixed it.